client/layout/guided-tours/tours/plugins-basic-tour.js

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { makeTour, Tour, Step, ButtonRow, Quit } from 'layout/guided-tours/config-elements';
+import { isEnabled } from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+import { getSelectedSite } from 'state/ui/selectors';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
+import { abtest } from 'lib/abtest';
+
+const isAtomic = state => {
+	const selectedSite = getSelectedSite( state );
```

Comment:
If we only use the site ID, why not `getSelectedSiteId` directly?

Reply:
Optional, but consider creating a new selector that does this, if we don't have it, say `state/selectors/is-selected-site-atomic`

Reply:
Yup, you're right. Let me update the code.